### PR TITLE
Adds action to web

### PR DESCRIPTION
**Pull request by williamlmao**
Sonner already supports `action` - adding parity on web.

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -100,6 +100,10 @@ export type BaseToastOptions = {
    */
   from?: "top" | "bottom";
   layout?: Layout;
+  action?: {
```
Comment (nandorojo) [inline src/types.ts:103]: Can you add a platform comment here

Comment (williamlmao) [reply src/types.ts:103]: Done!

**Comment by williamlmao** [comment]
hey @nandorojo - anything else you need from me here to get this merged?

File: src/types.ts (hunk 1 above)
Comment (nandorojo) [reply src/types.ts:103]: It’s resolved but the new code doesn’t show as pushed on my end

**Comment by nandorojo** [comment]
I think you need to push your changes

**Comment by williamlmao** [comment]
Sorry - I might just not understand merging into someone else's repo from my forked repo well enough. I have a 2nd commit in this PR that adds the comment. Are you unable to see that on your end?

SCR-20240228-oufb

**Comment by nandorojo** [comment]
Oh, you didn’t put the comment in the right place. It should be where I left my review and in the same format as the other comments.

**Comment by williamlmao** [comment]
@nandorojo ohhh I'm sorry about that - got confused between the two files since the keys were the same keys. Just opened a PR with a cleaner commit https://github.com/nandorojo/burnt/pull/41

**Pull request closed without merging**